docs/en/02_Developer_Guides/14_Files/03_File_Security.md

```diff
@@ -186,7 +182,7 @@ The urls for these two files, however, do not reflect the physical structure dir
   up the content of the hidden file, conditional on a permission check.
 
 When the file `NewCompanyLogo.gif` is made public, the url will not change, but the file location
```

Comment:
The URL use to serve the file does change. When serving a protected file, you will get a hash path.

Reply:
Reworded a bit